src/modules/launcher/PowerLauncher/PublicAPIInstance.cs

```diff
@@ -29,24 +26,28 @@ public class PublicAPIInstance : IPublicAPI, IDisposable
         private readonly SettingWindowViewModel _settingsVM;
         private readonly MainViewModel _mainVM;
         private readonly Alphabet _alphabet;
-        private readonly ThemeManager _themeManager;
         private bool _disposed;
 
         public event ThemeChangedHandler ThemeChanged;
 
-        public PublicAPIInstance(SettingWindowViewModel settingsVM, MainViewModel mainVM, Alphabet alphabet, ThemeManager themeManager)
+        public PublicAPIInstance(SettingWindowViewModel settingsVM, MainViewModel mainVM, Alphabet alphabet)
         {
             _settingsVM = settingsVM ?? throw new ArgumentNullException(nameof(settingsVM));
             _mainVM = mainVM ?? throw new ArgumentNullException(nameof(mainVM));
             _alphabet = alphabet ?? throw new ArgumentNullException(nameof(alphabet));
-            _themeManager = themeManager ?? throw new ArgumentNullException(nameof(themeManager));
-            _themeManager.ThemeChanged += OnThemeChanged;
+            ApplicationThemeManager.Changed += OnThemeChanged;
 
             ToastNotificationManagerCompat.OnActivated += args =>
             {
             };
         }
 
+        private void OnThemeChanged(ApplicationTheme currentApplicationTheme, Color systemAccent)
+        {
+            // oldTheme isn't used
+            ThemeChanged?.Invoke(currentApplicationTheme.ToTheme(), currentApplicationTheme.ToTheme());
```

Comment:
Done in order to not break plugins API: we should consider moving `ThemeChangedHandler` delegate to `Wox.Plugin` project and remove the old theme (not provided by WPF UI and not needed).